### ISSUE TITLE
Alpha order in collection browse is incorrect

### ISSUE THREAD

**Issue by laurensorensen** [sul-dlss/vt-arclight#457]
This is super minor, but all the other facets are in A-Z order except for Statements and Speeches - see screenshot. 
<img width="429" alt="Screen Shot 2023-01-10 at 3 34 44 PM" src="https://user-images.githubusercontent.com/3269689/211684370-38b6ee7c-219e-4e2a-9e0f-dbcf2fdf428b.png">

**Comment by ggeisler** [comment]
Good catch @laurensorensen. I agree that should be fixed. The alpha order is really nice and this stands out as not fitting in the right place.

**Comment by marlo-longley** [comment]
After a reindex this is now fixed on stage and prod.
